pkg/where-resolver/reconcile_on_edgeplacement.go

```diff
@@ -114,27 +116,43 @@ func (c *controller) reconcileOnEdgePlacement(ctx context.Context, epKey string)
 	}
 
 	// 4)
-	currentSPS, err := c.singlePlacementSliceLister.Cluster(epws).Get(epName)
+	_, originalName, kbSpaceID, err := kbuser.AnalyzeObjectID(ep)
 	if err != nil {
-		if errors.IsNotFound(err) { // create
+		logger.Error(err, "object does not appear to be a provider's copy of a consumer's object", "edgePlacement", ep.Name)
+		return err
```

Comment:
The fact that this func simply returns an error and all errors are treated the same (retry) has always been a problem and this PR exacerbates that.  Some errors are guaranteed to repeat on every retry and so should not be retried. This comment is on one of them.